clang-tools-extra/docs/ReleaseNotes.rst

```diff
@@ -48,6 +48,9 @@ Major New Features
 Improvements to clangd
 ----------------------
 
+- Introduced exmperimental support for C++20 Modules. The experimental support can
+  be enabled by `-experimental-modules-support` option.
```

Comment:
Done. I am only saying it is expected to not be efficient in real world cases. I didn't write the details since I think the major reader of release notes are end users.